WordPress/src/main/res/layout/add_content_bottom_sheet.xml

```diff
@@ -1,19 +1,52 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
+<RelativeLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
-    android:orientation="vertical"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:animateLayoutChanges="true">
+    android:layout_height="wrap_content">
 
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/content_recycler_view"
-        android:layout_height="match_parent"
+        android:layout_height="wrap_content"
         android:layout_width="match_parent"
         android:clipToPadding="false"
         android:descendantFocusability="beforeDescendants"
         android:scrollbars="vertical"
         android:layout_marginTop="@dimen/margin_extra_large"
         android:layout_marginBottom="@dimen/margin_extra_large"/>
 
-</LinearLayout>
+    <LinearLayout
```

Comment:
I'm not sure if it's applicable, but have you considered using `ActionableEmptyView`?

Reply:
My first implementation was using the `ActionableEmptyView` but @osullivanchris thought a `lighter` at least button and no image was fitting better in the context of the bottom sheet. Probably we could use the `ActionableEmptyView` styling in the wanted way the button of the view but for now I stayed with the simple layout as you have seen. Let me know if you feel we should change that in this PR 🙇‍♂️.